### DISCUSSION TITLE
Is it possible to run fastapi admin with sqlalchemy?

### DISCUSSION THREAD

**Discussion by KovalevVasiliy** [long2ice/fastapi-admin#117] [General]
Is it possible to run fastapi admin with sqlalchemy?

**Comment by MichaelNZ85** [comment]
Any update?

**Comment by phillipshaong** [comment]
Would love to have an admin portal that uses SQLModel, sqlalchemy.

**Comment by simonryf** [comment]
FastAPI Admin does only support TortoiseORM. If you need an admin Interface for FastAPI + SQLAlchemy you want to checkout [https://github.com/aminalaee/sqladmin](https://github.com/aminalaee/sqladmin)
